dynawo/sources/ModelicaCompiler/DYNCompilerModelicaOMC.cpp

```diff
@@ -150,11 +158,49 @@ int main(int argc, char ** argv) {
     std::cerr << " Compilation of " << modelName << " failed :" << s << std::endl;
   } catch (const char *s) {
     std::cerr << " Compilation of " << modelName << " failed :" << s << std::endl;
+  } catch (const DYN::Error& e) {
+    std::cerr << " Compilation of " << modelName << " failed :" << e << std::endl;
   } catch (...) {
     std::cerr << " Compilation of " << modelName << " failed : Unexpected exception " << std::endl;
   }
 }
 
+
+void
+prepareWorkspace(const string& modelName, const string& inputDir, const string& outputDir, bool& rmModels) {
+  if (!is_directory(outputDir))
+    create_directory(outputDir);
+  string outputDir1 = prettyPath(outputDir);
+  if (!is_directory(inputDir))
+    throw DYNError(DYN::Error::MODELER, MissingModelicaFile, absolute(modelName + ".mo", inputDir));
```

Comment:
I would have changed the error message to create a more specific one (even if it is correct, there is no Modelica model input but that is a bit worst -> there are no input data at the specified place)

Reply:
Solved